server/src/lib/dokku/plugin/database/create.ts

```diff
@@ -1,15 +0,0 @@
-import { NodeSSH } from 'node-ssh';
-
-export const create = async (
-  ssh: NodeSSH,
-  name: string,
-  databaseType: string
-) => {
```

Comment:
@Akirtovskis what do you think about keeping this file but accepting a 4th optional argument that will be the options passed down to `execCommand`? That way the command is still abstracted, so easier to reuse if we need later :)

Reply:
@pradel makes sense and probably worth to do the same for link and unlink as well. For this fn I will do in this PR, but for others create another to not make a mess out of it.

Reply:
Sounds good 👍